README.md

```diff
@@ -46,13 +47,24 @@ because for Mac, it requires performance boost with Docker Sync. Refer to
 `docker-compose.linux.yml` or `docker-compose.mac.yml` to `docker-compose.override.yml`
 in order to get your `ruby` workspace volumes mapped to your host machine.
 
+<a name="prerequisites"></a>
+### Prerequisites
+
+- Create `dockery` folder.
+
+```bash
+mkdir -p ~/.dockery/data/bundle/2.4.2
```

Comment:
Is it necessary to create this folder manually? I thought this should be handle by docker-compose.

Reply:
Apparently it is. This is in Mac.

Reply:
Seems like this issue only happens if we use `docker-sync`. I've updated `sync.sh` to include creating the necessary folders. I've also updated readme to notify user to use this script to start `docker-sync` service.